### PR TITLE
Refine props passed in advanced search params

### PR DESCRIPTION
**Pull request by syucream**
`SearchResultControlMenu` accepts filter conditions for **all** attributes but each component needs condition for specific attribute. I guess it occurs unnecessary re-rendering. So I refine props to get only necessary values.

### CODE REVIEW

File: frontend/src/components/entry/SearchResultControlMenu.tsx [hunk 1]
```diff
@@ -24,75 +24,61 @@ const StyledBox = styled(Box)({
 });
 
 interface Props {
-  attrName: string;
-  attrsFilter: AttrsFilter;
+  attrFilter: AttrFilter;
```
Comment (syucream) [inline frontend/src/components/entry/SearchResultControlMenu.tsx:27]: Here is the key point; changed prop type from `Record<K, V>` to `V`.

**Comment by hinashi** [comment]
https://github.com/dmm-com/airone/assets/5561786/41ab811a-8544-45d4-9037-ff744804ad96

However, it still took a long time to input.
This seems to occur when there are a large number of search results.
There are fewer results in the second half of the video, so it doesn't take much time to input.

The problem seems to be that the results are redrawn when the attrfilter state is updated.

**Comment by syucream** [comment]
@hinashi I have additional improvement on https://github.com/dmm-com/airone/pull/972/commits/20876321698a9e9f5be0ff5513da48bcc956de29, could you try it?

**Review by hinashi — APPROVED** [review]
LGTM
Thank you! very fast response

**Review by userlocalhost — APPROVED** [review]
Thank you very much for great improvement!! It's a good work 👍

**Pull request merged**